ui/src/components/credentials/ScanCredentialLink.tsx

```diff
@@ -192,29 +191,27 @@ export function ScanCredentialLink() {
           >
             <Space direction="vertical" size="large" style={{ maxWidth: "50vw" }}>
               <Typography.Title ellipsis={{ tooltip: true }} level={3}>
-                {shareCredentialQRCode.data.offerDetails.schemaTemplate.type}
+                {shareCredentialQRCode.data.linkDetail.schemaType}
               </Typography.Title>
 
               <Typography.Title level={5} type="secondary">
                 Attributes
               </Typography.Title>
 
-              {shareCredentialQRCode.data.offerDetails.attributeValues.map((attribute) => {
-                const formattedValue = formatAttributeValue(
-                  attribute,
-                  //TODO Credentials epic
-                  // shareCredentialQRCode.data.offerDetails.schemaTemplate.attributes
-                  []
-                );
+              {shareCredentialQRCode.data.linkDetail.attributes.map((attribute) => {
+                // const formattedValue = formatAttributeValue(
```

Comment:
Could you provide the PID for this?

Reply:
Are these two or just one of them?
* https://polygonid.atlassian.net/browse/PID-431
* https://polygonid.atlassian.net/browse/PID-437

Reply:
It depends on which part this is - those issues will both change based on what is decided to replace/amend them with. I would reference both here for now.

Reply:
Done